source/quick-start.txt

```diff
@@ -37,20 +37,25 @@ to check your version of Node and npm, as well as installation
 instructions, see `downloading and installing Node.js and npm 
 <https://docs.npmjs.com/downloading-and-installing-node-js-and-npm>`_.
 
+Select the :guilabel:`Clone the Project` tab if you want to build your
+project from a premade dependency file or the :guilabel:`Create the Project`
+tab if you want to create the project dependency file yourself.
+
 .. tabs::
 
    .. tab:: Clone the Project
       :tabid: clone the project
 
-      First, install Git using `Git's Getting Started Guide
+      First, ensure you have ``git`` installed in your development
+      environment or otherwise install Git using `Git's Getting Started Guide
```

Comment:
Comment: Having monospace and non-monospace "Git" in the same sentence was a bit odd for me. I'm not sure it increases clarity have the initial `git` in monospace, so might be worth changing. Non-blocking either way you go.

Reply:
I agree that it should be consistent across the same form when used outside of a title, so the instance within "otherwise install Git using" should be formatted similarly.
However, I was referring to git, the package name (see text-formatting in the style guide). I think I can just use the pronoun "it" in the aforementioned instance and leave the first instance as a monospaced package name.
Does that make sense?